docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lef/NangateOpenCellLibrary.macro.lef `as an example macro LEF file.
+* A Liberty file of the standard cell library with PVT characterization, input and output characteristics, timing and power definitions for each cell.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lib/NangateOpenCellLibrary_typical.lib `as an example liberty file.
+* For KLayout: A mapping from LEF/DEF to GDS layers:datatypes
+
+Adding a new platform additionally requires the following: 
+
+* A validated installation of the OpenROAD flow scripts is available. See instructions [here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html#building-and-installing-the-software).:
+* A general knowledge of VLSI design and RTL to GDS flows.  \
+OpenROAD implements a fully-automated RTL-GDSII but it requires familiarity with the OpenROAD flow scripts flow to enable debugging in case of problems.
+
+
+# Adding a New Platform to OpenROAD
+
+
+## Setup
+
+This section describes the necessary files and  directories needed to build the platform.  All files and directories made/edited are independent of each other unless otherwise stated.
+
+
+## Makefile
+
+Make the following edits  to the Makefile (located in `OpenROAD-flow-scripts/flow/Makefile`) so that OpenROAD can run the flow on a design using the new platform.
+
+1. At  the beginning of the Makefile, there is a block of `DESIGN_CONFIG` variables that are commented out. These variables tell OpenROAD which design to run and on what platform. `DESIGN_CONFIG` specifically points to a `config.mk` file located in the designs directory for the respective platform. A `DESIGN_CONFIG` variable must be added for the new platform using a specific design. OpenROAD has multiple Verilog designs already made which can be used with any platform (see `OpenROAD-flow-scripts/flow/designs/src` for a list of usable designs). For example, a `DESIGN_CONFIG` variable using the `riscv32i` design on a new platform would look as follows:
+
+
+```
+DESIGN_CONFIG=./designs/MyNewPlatform/riscv32i/config.mk
+```
+
+
+The `config.mk` file will be generated later in the [Design Directory](#DesignDirectory) section of this document.
+
+
+## Platform Directory
+
+Create a directory for the new technology inside `OpenROAD-flow-scripts/flow/platforms` to contain the necessary files for the OpenROAD flow.
+
+
+```
+$ mkdir OpenROAD-flow-scripts/flow/platforms/MyNewPlatform
+```
+
+
+
+## Design Directory
+
+The design directory contains the configuration files for all the designs of a specific platform. Create a directory for the new platform in OpenROAD-flow-scripts/flow/designs to contain the relevant files and directories for all the designs for the flow in that specific platform. Each design requires its own `config.mk` and `constraints.sdc` files. 
+
+Follow the steps below to create the necessary directories and files. **NOTE: riscv32i is just an example and not a required name.**:
```

Comment:
I would recommend replacing riscv32i with gcd. If users follow these example instructions, they should be using the simplest design we have.

Reply:
Replaced all examples with riscv32i to gcd